ballerina/Ballerina.toml

```diff
@@ -10,16 +10,16 @@ license = ["Apache-2.0"]
 distribution = "2201.8.0"
 export = ["protobuf", "protobuf.types.any", "protobuf.types.duration", "protobuf.types.empty", "protobuf.types.struct", "protobuf.types.timestamp", "protobuf.types.wrappers"]
 
-[platform.java17]
+[platform.java21]
 graalvmCompatible = true
 
-[[platform.java17.dependency]]
+[[platform.java21.dependency]]
 groupId = "io.ballerina.stdlib"
 artifactId = "protobuf-native"
 version = "1.6.2"
```

Comment:
Should we do this?
```suggestion
version = "1.6.3"
```

Reply:
We can do when we merge final changes to master.